src/test/lut.test.js

```diff
@@ -144,6 +144,42 @@ describe("test histogram", () => {
       expect(secondlut.lut[1 * 4 + 3]).to.eql(255);
       expect(lut.lut[255 * 4 + 3]).to.eql(255);
       expect(secondlut.lut[255 * 4 + 3]).to.eql(255);
+      // Make sure no NaN values
+      lut.controlPoints.forEach(controlPoint => {
+        expect(controlPoint.opacity).to.be.finite;
+      })
+    });
+    it("is consistent for minMax edge case 120,120", () => {
```

Comment:
instead of calling this "edge case 120,120" could you put what makes this an edge case in the description? Same for the tests that follow

Reply:
Good call. I made the descriptions more clear and grouped some tests together. (see new groups `lutGenerator_minMax` and `lutGenerator_windowLevel`)

![image](https://user-images.githubusercontent.com/12690133/135920431-bd731cbb-9556-4445-a2fb-2f09d8446484.png)